wnfs/examples/privateref.rs

```diff
@@ -10,13 +10,16 @@ use wnfs::{
     utils, Hasher, MemoryBlockStore, Namefilter, PrivateDirectory, PrivateOpResult,
 };
 
-#[async_std::main]
+#[tokio::main(flavor = "multi_thread")]
```

Comment:
Any particular reason? Or mainly acknowledging that it's the more 'common' async runtime today?

Reply:
Just thought to try another runtime.